### ISSUE TITLE
[bug] Panic on 32-bit ARM with Centrifugo v2.6.x

### ISSUE THREAD

**Issue by FZambia** [centrifugal/centrifugo#387]
**Describe the bug**
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11f08]

goroutine 50 [running]:
runtime/internal/atomic.goStore64(0x14be3fc, 0x1fff, 0x0)
        /home/travis/.gimme/versions/go1.14.7.linux.amd64/src/runtime/internal/atomic/atomic_arm.go:144 +0x1c
github.com/centrifugal/centrifuge.(*Client).connectCmd(0x14be3c0, 0x191a1e0, 0x1914078, 0x70)
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/client.go:1243 +0x118
github.com/centrifugal/centrifuge.(*Client).handleConnect(0x14be3c0, 0x191c0e0, 0x6c, 0x70, 0x1914078, 0x1922014)
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/client.go:839 +0x1a4
github.com/centrifugal/centrifuge.(*Client).handleCommand(0x14be3c0, 0x191a100, 0x191a180, 0x191a1a0, 0x151f7d8)
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/client.go:738 +0x440
github.com/centrifugal/centrifuge.(*Client).Handle(0x14be3c0, 0x1920000, 0x7e, 0x600, 0x0)
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/client.go:646 +0x398
github.com/centrifugal/centrifuge.(*WebsocketHandler).ServeHTTP.func2.3(0x142e7e0, 0x14be3c0, 0x1920000, 0x7e, 0x600, 0x1562058)
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/handler_websocket.go:353 +0x40
created by github.com/centrifugal/centrifuge.(*WebsocketHandler).ServeHTTP.func2
        /home/travis/gopath/pkg/mod/github.com/centrifugal/centrifuge@v0.10.1/handler_websocket.go:349 +0x468
```

…

**Versions**
> Centrifugo version is 2.6.1
> -
> Operating system is Raspbian 32 bit

...

**Steps to Reproduce**
> Connect to Centrifugo

**Comment by FZambia** [comment]
The reason already found – this is wrong atomic aligment inside struct, will make a release with fix very soon.

**Comment by FZambia** [comment]
Fix: https://github.com/centrifugal/centrifuge/commit/cafa94fbf4173ae46d1f5329a33adec97d0620c8

**Comment by FZambia** [comment]
Fix released in v2.6.2
